### PR TITLE
Bump to Rails 5.0.0.beta4

### PR DESCRIPTION
**Pull request by prathamesh-sonpatki**

### CODE REVIEW

**Comment by prathamesh-sonpatki** [comment]
cc @schneems

**Comment by prathamesh-sonpatki** [comment]
Merging this

File: config/spring.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+%w(
```
Comment (schneems) [inline config/spring.rb]: Can we remove spring from our gemfile instead? 🔥 🔥 🔥

**Comment by schneems** [comment]
Thanks!

File: config/spring.rb (hunk 1 above)
Comment (prathamesh-sonpatki) [inline config/spring.rb]: :smile: Sure I will do that

**Pull request merged**